packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -120,3 +121,22 @@ $swatch-gap: 12px;
 		flex-shrink: 0;
 	}
 }
+
+.block-editor-panel-color-gradient-settings__reset {
+	position: absolute;
+	display: inline-flex;
+	right: 0;
+	top: 0;
+	bottom: 0;
```

Comment:
I feel like `bottom: 0` shouldn't be necessary if we have the proper size and `top` and `right` specified.